cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -62,7 +62,7 @@ type VMTServer struct {
 	UseVMWare bool
 
 	// Kubelet related config
-	KubeletPort        uint
+	KubeletPort        int
```

Comment:
I looked the code of Heapster and [Kubernetes](https://github.com/songbinliu/kubernetes/blob/6f7eac63c2d170100ef95f65aa2d57f9812243f4/pkg/apis/componentconfig/types.go#L201). They are all using Int32 as port type.

I guess the reason is that uint is more problemsome. For example, if we use uint as port number, someone may want to convert a Integer value (-10) to uint (4294967286), and assign to the port. This will cause more undetectable problem.

Reply:
I think the port number cannot be larger than 65536. Don't know why `int32` is used there.